joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
+In contrast, Phylotrack provides ready-built tracking flexible enough to attach to any simulation of replicating entities.
 
-Two other general-purpose libraries for recording phylogenetic information do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
-However, they allow different modes of phylogenetic tracking than Phylotrack does.
+Two other general-purpose libraries for phylogenetic tracking do exist: hstrat and Automated Phylogeny Over Geological Timescales (APOGeT).
+However, they pursue alternate modes of record keeping to Phylotrack.
 Whereas Phylotrack uses a graph-based approach to perfectly record asexual phylogenies, the hstrat library implements hereditary stratigraphy, a recently developed method that allows robust decentralized phylogenetic tracking in parallel and distributed systems at the cost of a tunable reduction in accuracy [@moreno2022hstrat].
 APOGeT, in turn, focuses on tracking speciation in sexually-reproducing populations [@godin2019apoget].
 
 Vast amounts of bioinformatics-oriented phylogenetics software is also available.
 These programs' purposes typically include
-- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic]
-- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating]
-- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic]
-- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython]
+- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic],
+- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating],
+- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic], and
+- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython].
 
 Phylotrack overlaps with these goals only in that it also provides tree statistic implementations. We chose to include this feature to facilitate fast during-simulation calculations of these metrics.
-Notably, the problem of tracking a phylogeny within an agent-based program is substantially different from the more traditional problem of reconstructing a phylogeny
-(indeed, users new to working with recorded phylogenies should refer to the Phylotrackpy documentation for notes on subtle structural differences from reconstructed phylogenies).
+Notably, the problem of tracking a phylogeny within an agent-based program is substantially different from the more traditional problem of reconstructing a phylogeny.
+(Indeed, users new to working with recorded phylogenies should refer to the Phylotrackpy documentation for notes on subtle structural differences from reconstructed phylogenies.)
```

Comment:
Aren't you not supposed to have stand-alone parenthetical sentences like this?

Reply:
I think it's ok? On a reread, wondering if we should nix indeed.

https://apastyle.apa.org/learn/faqs/use-parentheses
https://style.mla.org/how-to-use-parentheses/

Reply:
Actually, maybe we should just drop both the indeed and the parentheses

Reply:
agreed here